allennlp/tests/commands/train_test.py

```diff
@@ -251,6 +251,21 @@ def test_train_model_can_instantiate_from_params(self):
             params=params, serialization_dir=self.TEST_DIR, local_rank=0, batch_weight_key=""
         )
 
+    def test_train_can_fine_tune_model_from_archive(self):
+        params = Params.from_file(
+            self.FIXTURES_ROOT / "basic_classifier" / "experiment_from_archive.jsonnet"
+        )
+        train_loop = TrainModel.from_params(
+            params=params, serialization_dir=self.TEST_DIR, local_rank=0, batch_weight_key=""
+        )
+        train_loop.run()
+
+        # This is checking that the vocabulary actually got extended.  This token was not in the
+        # original training data for the model that we are fine-tuning, but is in the new training
+        # data, and the vocabulary settings in the experiment config are such that this token should
+        # have been added.
+        assert "journalism" in train_loop.model.vocab.get_token_to_index_vocabulary()
```

Comment:
Instead of this test relying on the data, could you just compare the vocab size or something? Is it guaranteed to be larger?

Reply:
Hmm, ok, yeah, I could do that.  I'll have to load the original model again separately, which I hadn't thought of doing before, but it should work just fine.